test/e2e/tests/confirmations/transactions/transaction-decoding-redesign.spec.ts

```diff
@@ -171,6 +173,7 @@ async function mocked4BytesResponse(mockServer: MockttpServer) {
     // TODO: Fix in https://github.com/MetaMask/metamask-extension/issues/31860
     // eslint-disable-next-line @typescript-eslint/naming-convention
     .withQuery({ hex_signature: '0x3b4b1381' })
+    .always()
```

Comment:
This is to stabilize the test because I've seen [1 failure ](https://github.com/MetaMask/metamask-extension/actions/runs/18686651653/job/53282317538?pr=37013)because of this mock not being applied, rather the one we have in the mock-e2e globals.
This can happen if 2 requests are sent (the first one is captured by the custom mock the 2nd one with our global mocks)
It's seems a very rare race condition and I cannot repro manually, so I haven't opened a bug ticket for now.

<img width="400" height="619" alt="image" src="https://github.com/user-attachments/assets/7a6d8c37-be2a-4fd5-816e-a66eb0300503" />